deployments/registry.go

```diff
@@ -129,44 +132,64 @@ func (k Registry) apply(ctx context.Context, c *kubernetes.Cluster, ui *termui.U
 		return err
 	}
 
+	log.Info("extracting chart file", "name", registryChartFile)
 	tarPath, err := helpers.ExtractFile(registryChartFile)
 	if err != nil {
 		return errors.New("Failed to extract embedded file: " + tarPath + " - " + err.Error())
 	}
 	defer os.Remove(tarPath)
 
+	log.Info("local transient tar archive", "name", tarPath)
+
 	htpasswd, err := registryAuth.Htpassword()
 	if err != nil {
 		return errors.Wrap(err, "Failed to hash credentials")
 	}
 
+	log.Info("htpasswd from credentials", "htpasswd", htpasswd)
+
 	domain, err := options.GetString("system_domain", TektonDeploymentID)
 	if err != nil {
 		return errors.Wrap(err, "Couldn't get system_domain option")
 	}
 
+	log.Info("system domain", "domain", domain)
+
+	var helmArgs []string
+
+	log.Info("assembling helm command")
 	// (**) See also `deployments/tekton.go`, func `createClusterRegistryCredsSecret`.
-	helmCmd := fmt.Sprintf("helm %[1]s %[2]s --set 'auth.htpasswd=%[3]s' --set 'domain=%[4]s' --set 'createNodePort=%[5]v' --namespace %[6]s %[7]s",
-		action,
-		RegistryDeploymentID,
-		htpasswd,
-		fmt.Sprintf("%s.%s", RegistryDeploymentID, domain),
-		options.GetBoolNG("use-internal-registry-node-port"),
-		RegistryDeploymentID,
-		tarPath)
-	if out, err := helpers.RunProc(helmCmd, currentdir, k.Debug); err != nil {
+
+	helmArgs = append(helmArgs, action, RegistryDeploymentID)
+	helmArgs = append(helmArgs, `--namespace`, RegistryDeploymentID)
+	helmArgs = append(helmArgs, tarPath)
+	helmArgs = append(helmArgs, `--set`, `auth.htpasswd=`+htpasswd)
+	helmArgs = append(helmArgs, `--set`, fmt.Sprintf("domain=%s.%s", RegistryDeploymentID, domain))
+	helmArgs = append(helmArgs, `--set`, fmt.Sprintf(`createNodePort=%v`, options.GetBoolNG("use-internal-registry-node-port")))
```

Comment:
This is the second time I see this, couldn't this  just be an array assignment?

`args := []string{"helm", "--set", "...", "--set", ...}`

Anyhow, not important.

Reply:
I kind of need the changes to `acceptance/helper/epinio/Epinio`, so I'll use this branch as a base then

Reply:
Done with commit 51460961683b0bc98df5e5bcb40a074708b872c0